validators/notes.go

```diff
@@ -31,7 +31,7 @@ func ValidateUpdateGroupNote(in *accountsv1.UpdateGroupNoteRequest) error {
 func ValidateListGroupNote(in *accountsv1.ListGroupNotesRequest) error {
 	err := validation.ValidateStruct(in,
 		validation.Field(&in.GroupId, validation.Required, is.UUID),
-		validation.Field(&in.AuthorAccountId, validation.Required, is.UUID),
+		validation.Field(&in.AuthorAccountId, validation.When(in.AuthorAccountId != "", validation.Required), is.UUID),
```

Comment:
> Required is a validation rule that checks if a value is not empty. A value is considered not empty if - integer, float: not zero - bool: true - string, array, slice, map: len() > 0 - interface, pointer: not nil and the referenced value is not empty - any other types
```suggestion
		validation.Field(&in.AuthorAccountId, is.UUID),
```
What you're doing here is a noop I think.